tests/e2e/integration/tip.js

```diff
@@ -40,7 +40,7 @@ describe('Test cases for tip page', () => {
       .enterTipDetails({ amount: tip.amount })
       .buttonShouldBeDisabled('[data-cy=send-tip]')
       .enterTipDetails({ note: tip.note })
-      .buttonShouldNotBeDisabled('[data-cy=send-tip]')
+      .buttonShouldBeDisabled('[data-cy=send-tip]')
```

Comment:
I think it should be disabled because user before was changing amount and left it invalid (empty) 🤷‍♀️